src/views/components/model.jsx

```diff
@@ -40,18 +40,17 @@ export default class Model extends React.Component {
                         <p>{this.state.model.description}</p>
                     </div>
                     <div className="faultArea">
-                        <h3>Fault</h3>
-                        {(this.state.model.isFaulty
-                                ? <p>{this.state.model.faultDescription}</p>
-                                : <p>Model is not faulty.</p>
+                        {(this.state.model.allowCheckout
+                                ? <span><b>Total:</b> {this.state.model.count}<br/><b>In Stock:</b> {this.state.model.inStock}</span>
+                                : <i>Model cannot be checked out.</i>
                         )}
                     </div>
                     <div className="miscArea">
                         <b>Location:</b> {this.state.model.location}<br/>
                         <b>Price:</b> ${this.state.model.price}<br/>
                         <b>Manufacturer:</b> {this.state.model.manufacturer}<br/>
                         <b>Vendor:</b> {this.state.model.vendor}<br/>
-                        <b>Quantity:</b> {this.state.model.count}<br/>
+                        {(this.state.model.allowCheckout ? null : <span><b>Quantity:</b> {this.state.model.count}<br/></span>)}
```

Comment:
You've flipped your logic between line 53 and 43.
This view really doesn't make sense anymore. If the model is not serialized(allowCheckout = true) then you display the total and the amount they have in stock. If it's not serialized(allowCheckout = false) you don't display the Total/InStock but that concept still exists for models that are not serialized. Their Total is their quantity(count) and their InStock is the amount of items they have. 
You should probably rethink how to display this information to the user and what you are trying to tell them with this information.